static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -0,0 +1,65 @@
+# Updating pipelines
+
+If you need to rename or relocate a DVC-file and/or dependencies and outputs
+of a specific DVC-file, then you can
+
+- use `dvc move` to edit file paths of outputs or dependencies.
```

Comment:
there should be a note immediately that it's only about `dvc added` stages. They don't have dependencies. Also outputs are not always files.